### PR TITLE
added p-value correction for geom_signif

### PR DESCRIPTION
**Pull request by Insaynoah**
I added the p-value correction to geom_signif in the boxplot example of chapter 6 as specified in #340.
I also added the missing packages in description as stated in #406.

### CODE REVIEW

File: inst/pages/14_alpha_diversity.qmd [hunk 1]
```diff
@@ -134,11 +134,13 @@ comb <- split(t(combn(levels(df$SampleType), 2)),
...
 ggplot(df, aes(x = SampleType, y = shannon)) +
-  # Outliers are removed, because otherwise each data point would be plotted twice; 
+  # Outliers are removed, because otherwise each 
+  # data point would be plotted twice; 
   # as an outlier of boxplot and as a point of dotplot.
   geom_boxplot(outlier.shape = NA) + 
   geom_jitter(width = 0.2) + 
-  geom_signif(comparisons = comb, map_signif_level = FALSE) +
+  geom_signif(comparisons = comb, map_signif_level = FALSE,
+              correction="holm") + #corrects the p-values
```
Comment (antagomir) [inline inst/pages/14_alpha_diversity.qmd:143]: could you change this to "fdr", that is more common default

**Comment by antagomir** [comment]
Thanks! Just a minor edit proposed in the comment.

**Comment by Insaynoah** [comment]
done

**Comment by antagomir** [comment]
Thanks. I let @TuomasBorman do the final squash / merges.

**Pull request merged**